adding_species_manuscript.tex

```diff
@@ -525,53 +523,59 @@ \subsection*{Basic setup for chromosome-level simulations}
+  maintenance and storage burden incurred by the large number of contigs in these partial assemblies (see also discussion below).
 
 \item
   \textbf{An average mutation rate} for each chromosome (per generation per bp).
   This rate estimate can be based on sequence data from pedigrees, mutation accumulation studies, 
   or comparative genomic analysis calibrated by fossil data (i.e., phylogenetic estimates).
-  %If none of these are available for your species of interest, you may use an estimate obtained for another, closely related, species.
-  Although mutation rates {Benzer1961,Ellegren2003} and processes {Supek2019} are not uniform along the genome or through time,
-  at present mutations are simulated at a constant rate under the Jukes-Cantor model of nucleotide mutations (CITE).
-  We antcipate future efforts will provide support for more complex, heterogeneous mutational processes,
-  as these are easily specified in both the SLiM and msprime simulation engines.
+  At present, \stdpopsim simulates mutations at a constant rate under the Jukes-Cantor model of nucleotide mutations \citep{Jukes1969}.
+  However, we anticipate future development will provide support for more complex, heterogeneous mutational processes,
+  as these are easily specified in both the \texttt{SLiM} and \texttt{msprime} simulation engines.
+  Such progress will further improve the realism of simulated genomes,
+  since mutation rates and processes are known to vary along the genome and through time \citep{Benzer1961,Ellegren2003,Supek2019}.
 
 \item
   \textbf{Recombination rates} (per generation per bp).
   Ideally, a population genomic simulation should make use of a chromosome-level \textbf{recombination map}, 
-  since the recombination rate is known to vary widely across chromosomes {Nachman2002},
+  since the recombination rate is known to vary widely across chromosomes \citep{Nachman2002},
   and this can strongly affect the patterns of linkage disequilibrium and shared haplotype lengths.
   When this information is not available, we suggest specifying an average recombination rate for each chromosome.
   At minimum, an average genome-wide recombination rate needs to be specified, which is typically available for well assembled genomes.
-  %As with the average mutation rate, if an estimate is not available for your species of interest, you may use an estimate obtained for another, closely related, species.
+  %ILAN: added this part about GC
+  Recall that for bacteria and archea, which primarily experience recombination by gene conversion,
+  the recombination rate corresponds to the rate of gene conversion,
+  and the average tract length should also be specified
+  (see details in previous section).
+  If one wishes to model gene conversion together with crossover recombination,
+  then they should specify the fraction of recombinations done by gene conversion
+  as well as the average tract length (per chromosome).
```

Comment:
People can still add GC rates themselves to contigs for which we haven't specified GC rates, but this is no longer part of the standard API - perhaps this should say instead just something saying that GC is available but optional.

Reply:
I didn't get your point here. I want to stick to the framework as we describe it in the gene conversion paragraph. One may model GC by specifying for each chrom a % of recs done by GC + an average tract length. Do we want to say anything more here?

Reply:
I think adding a subsection for gc in eukaryotes might be a good way to disentangle it here.

E.g. starting from Line 553:

\textbf{gene conversion (optional)}
If one wishes to model gene conversion in eukaryotes either together with crossover recombination or as a stand-alone process,
  then they should specify the fraction of recombinations done by gene conversion
  as well as the average tract length (per chromosome).